woocommerce/woocommerce-ios
# [Shipping Labels] Edge-to-edge table views

Part of #3716 

# Description
This PR aims to update table views in Shipping Labels forms to extend edge-to-edge.

# Changes
## UIKit views: Address Form and Suggested Address 
- Updated leading and trailing edges for table views to constrain to their super views' edges.
- Pinned top banner view to all edges of the table views' headers.
- Pinned content of top banner view to safe areas.

## SwiftUI views: Package details, package list, Carriers and Payment methods
- Added a new view modifier to add paddings to a view based on input insets and a set of edges.
- Ignore safe areas on the scroll views.
- Retrieved safe area insets from GeometryReader and use them to add paddings to table contents.
- Updated top banner wrapper view to accept edge insets to use as paddings for the top banner view.
- Sent safe area insets to banner views of Carriers and Payment Methods screens to keep contents within safe areas.

# Screenshots
| Before | After |
| ------ | ------ |
| <img src="https://user-images.githubusercontent.com/5533851/126337898-512b5565-446a-4e22-a23c-1ee64f4778c7.png" width=400 /> | <img src="https://user-images.githubusercontent.com/5533851/126336204-73beccc0-90a3-4db2-a571-03d9d486f73e.png" width=400 /> |
| <img src="https://user-images.githubusercontent.com/5533851/126337945-264dedd8-660a-4fef-8dd1-688f086455d3.png" width=400 /> | <img src="https://user-images.githubusercontent.com/5533851/126336254-553d9dd4-fd3f-4e5b-b779-4ff66ac6de43.png" width=400 /> |
| <img src="https://user-images.githubusercontent.com/5533851/126338038-bdfe865f-96cc-4d19-81d7-f7243a6335ae.png" width=400 /> | <img src="https://user-images.githubusercontent.com/5533851/126336329-2b1b3c70-2062-4f16-9e93-3ca6cdd951bf.png" width=400 /> | 
| <img src="https://user-images.githubusercontent.com/5533851/126338091-5eb8c74c-a3d6-461c-b601-f006279300f9.png" width=400 /> | <img src="https://user-images.githubusercontent.com/5533851/126336382-e16fe76c-4c1d-4e40-b232-90498dbeae1c.png" width=400 /> | 
| <img src="https://user-images.githubusercontent.com/5533851/126338135-0cd684a2-49e4-4a6e-b888-484f0041729b.png" width=400 /> | <img src="https://user-images.githubusercontent.com/5533851/126336444-c36059a5-aad0-4c3a-9af7-1be205299874.png" width=400 /> | 
| <img src="https://user-images.githubusercontent.com/5533851/126338167-ec836242-26ea-4fba-b675-8829b507637d.png" width=400 /> | <img src="https://user-images.githubusercontent.com/5533851/126336498-8a3fd005-6256-4cab-8446-25b5a54fc5b2.png" width=400 /> | 
| <img src="https://user-images.githubusercontent.com/5533851/126338175-263d1f04-b9c1-4158-8d2e-78ed2eba5856.png" width=400 /> | <img src="https://user-images.githubusercontent.com/5533851/126336555-f01c1d11-5b70-48b9-94f4-ac8b0a9af080.png" width=400 /> |

# Testing
1. Switch to a store that's eligible for creating shipping labels.
2. Navigate to Orders tab.
3. Select an order with Processing status.
4. Select Create Shipping Label.
5. Navigate through each step of the form in portrait mode - notice that layout is not broken.
6. Switch to landscape mode and navigate through each step again - notice that the table views and banner views all stretch to edges of the screen while contents stay in safe areas (not clipped by iPhone notch).

Update release notes:
